snap/local/files/run_dcgm_exporter.sh

```diff
@@ -4,11 +4,18 @@ set -euo pipefail
 # Build the argument list for the dcgm-exporter command
 args=()
 
-# Add the dcgm-exporter-address option if it is set. Default: “:9400”
+# Add the dcgm-exporter-address option if it is set.
 dcgm_exporter_address="$(snapctl get dcgm-exporter-address)"
+# Add the dcgm-exporter-metrics-file option if it is set.
+dcgm_exporter_metrics_file_path="$SNAP_COMMON/$(snapctl get dcgm-exporter-metrics-file)"
 
 if [ -n "$dcgm_exporter_address" ]; then
     args+=("-a" "$dcgm_exporter_address")
 fi
 
+# File should be available in the snap data directory under $SNAP_COMMON
+if [[ -f "$dcgm_exporter_metrics_file_path" && -s "$dcgm_exporter_metrics_file_path" ]]; then
```

Comment:
Do we need to add the `-s` condition?  Just checking if the file exists should be enough.

Reply:
Also note that this validation is silent to the user - if they set a value but there's a typo in the file path, they won't know.

Reply:
We don't want to provide empty files to the `dcgm-exporter` as it will behave the same as just turning the exporter off